pubspec.yaml

```
@@ -15,7 +15,7 @@ dependencies:
   flutter:
     sdk: flutter
   http: ">=0.13.1 <2.0.0"
-  lints: ">=1.0.1 <2.1.2"
+  lints: ^3.0.0
```

```suggestion
  lints: ">=1.0.1"
```

This may cause conflicts if another package has an upper bound for `lints`.